pkg/repository/prisma/buffer_add_mq.go

```diff
@@ -0,0 +1,83 @@
+package prisma
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hatchet-dev/hatchet/pkg/repository/buffer"
+	"github.com/hatchet-dev/hatchet/pkg/repository/prisma/dbsqlc"
+	"github.com/hatchet-dev/hatchet/pkg/repository/prisma/sqlchelpers"
+
+	"github.com/jackc/pgx/v5/pgtype"
+)
+
+func newAddMQBuffer(shared *sharedRepository) (*buffer.TenantBufferManager[addMessage, int], error) {
+	userEventBufOpts := buffer.TenantBufManagerOpts[addMessage, int]{
+		Name:       "add_mq",
+		OutputFunc: shared.bulkAddMessages,
+		SizeFunc:   sizeOfMQMessage,
+		L:          shared.l,
+		V:          shared.v,
+	}
+
+	manager, err := buffer.NewTenantBufManager(userEventBufOpts)
+
+	if err != nil {
+		shared.l.Err(err).Msg("could not create tenant buffer manager")
+		return nil, err
+	}
+
+	return manager, nil
+}
+
+func sizeOfMQMessage(item addMessage) int {
+	return len(item.payload)
+}
+
+type addMessage struct {
+	queue   string
+	payload []byte
+}
+
+func (r *sharedRepository) bulkAddMessages(ctx context.Context, opts []addMessage) ([]*int, error) {
+	res := make([]*int, 0, len(opts))
+	p := []dbsqlc.BulkAddMessageParams{}
+
+	for index, opt := range opts {
+		i := index
+		res = append(res, &i)
+
+		p = append(p, dbsqlc.BulkAddMessageParams{
+			QueueId: pgtype.Text{
+				String: opt.queue,
+				Valid:  true,
+			},
+			Payload:   opt.payload,
+			ExpiresAt: sqlchelpers.TimestampFromTime(time.Now().UTC().Add(5 * time.Minute)),
```

Comment:
nit: pass interval instead of timestamp so clock is based on db since we're using now() for delete

Reply:
The issue is `copyfrom`: `:copyfrom doesn't support non-parameter values`